packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-node.vue

```diff
@@ -97,6 +95,11 @@ watch(
 		const modelId = inputs.find((input) => input.type === 'modelId')?.value?.[0];
 		const state = cloneDeep(props.node.state);
 
+		const modelInputs = inputs.filter((input) => input.type === 'modelId');
+		if (modelInputs[0].status === WorkflowPortStatus.CONNECTED) {
+			isModelInputConnected.value = true;
+		}
```

Comment:
Is this the only reason the `immediate: true` is at the end of the watcher? Maybe this should be in an `onMounted`?

Reply:
This needs to be watched because the connection can change. 

The reason of immediate is to compensate for what was previously a semi-computed isModelInputConnected variable which, as far as I know, wasn't hooked up correctly. Model-config, likewise has the same kind of scheme and I made the same corresponding change.